pypowsybl/network.py

```diff
@@ -4124,6 +4124,39 @@ def remove_extensions(self, extension_name: str, ids: _Union[str, _List[str]]) -
             ids = [ids]
         _pp.remove_extensions(self._handle, extension_name, ids)
 
+    def get_connectables_order_positions(self, voltage_level_id: str) -> _DataFrame:
+        """
+        Gets the order positions of every connectable of a given voltage level in a dataframe.
```

Comment:
I think we should go into more details about what is the "order position" (in the long description, so after the first line): 
what it is, where it used.